WooCommerce/Classes/ViewRelated/Orders/Order Details/Customer Section/Billing Information/BillingInformationViewController.swift

```diff
@@ -388,7 +388,10 @@ private extension BillingInformationViewController {
             }
 
             let title = NSLocalizedString("Contact Details", comment: "Section header title for contact details in billing information")
-            return Section(title: title, secondaryTitle: nil, rows: rows)
+            if rows.count ===0 {
```

Comment:
It would be great for this use case, to use the `guard` statement instead of an `if` condition here.